gateway/src/main/java/io/zeebe/gateway/impl/broker/BrokerRequestManager.java

```diff
@@ -115,37 +116,57 @@ private static boolean responseValidation(final DirectBuffer responseContent) {
       nodeIdProvider = determineBrokerNodeIdProvider(request);
     } catch (final PartitionNotFoundException e) {
       returnFuture.completeExceptionally(e);
+      GatewayMetrics.registerFailedRequest(request.getPartitionId(), request.getType());
```

Comment:
In this case, the request was never send to a broker. Do we want to include them in the metrics?

Reply:
Why not? My reasoning was that it's still a failure on the gateway's part so it'd be useful to include it. But we can be more strict with what we consider a failure, if that allows us to extract more value from the graph. What's your opinion?

Reply:
That makes sense. May be, as I mentioned in another comment, we can add the reasons for failures to the metrics. So when we want to get more detailed insight, we can filter out irrelevant failures.

Reply:
I agree, I'll add that and post some screen shots of how it looks